### ISSUE TITLE
do you have plan to dump watchOS 2 headers

### ISSUE THREAD

**Issue by NilStack** [nickfrey/WatchOS-Headers#2]
Hi, Nick:

Thanks for the work on watchOS header. 
do you have plan to dump watchOS 2 headers?

Thanks

**Comment by nickfrey** [comment]
Yes, I do. Just haven't gotten around to it yet – although should soon! Thanks for filing an issue on it.

**Comment by NilStack** [comment]
Thanks for your reply, Nick.

**Comment by Baddaboo** [comment]
Any progress on this?
